pkg/features/versioned_kube_features.go

```diff
@@ -681,6 +681,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	StatefulSetAutoDeletePVC: {
 		{Version: version.MustParse("1.23"), Default: false, PreRelease: featuregate.Alpha},
 		{Version: version.MustParse("1.27"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.32, remove in 1.34
```

Comment:
Based on the new guideline (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/feature-gates.md#compatibility-versions) we'll remove it only in 1.35.

Reply:
done